mteb/abstasks/TaskMetadata.py

```diff
@@ -137,6 +140,14 @@
     "i2it",  # image-to-image+text
     "t2it",  # text-to-image+text
     "it2it",  # image+text-to-image+text
+    "a2a",  # audio to audio
+    "a2t",  # audio to text
+    "t2a",  # text to audio
+    "at2t",  # audio+text-to-text
+    "at2a",  # audio+text-to-audio
+    "a2at",  # audio-to-audio+text
+    "t2at",  # text-to-audio+text
+    "at2at",  # audio+text-to-audio+text
```

Comment:
Audio multilabel classification should be "a2t", but I'm ok to leave this as is.